src/Illuminate/Support/Enumerable.php

```diff
@@ -18,6 +18,14 @@ interface Enumerable extends Arrayable, Countable, IteratorAggregate, Jsonable,
      */
     public static function make($items = []);
 
+    /**
+     * Create a new collection instance from the characters in the given string.
+     *
+     * @param  string  $string
+     * @return static
+     */
+    public static function fromString($string);
```

Comment:
Breaking change.

Reply:
How so?

Reply:
I guess if someone is implementing `Enumerable` themselves somewhere.

Reply:
I removed it from the contract.